_posts/2018-09-06-how-we-use-pivotal-tracker-at-ombu-labs.markdown

```diff
@@ -2,155 +2,121 @@
-can estimate it with 0, 1, 2, 3, 5, or 8 points. We prefer to use a [Fibonacci](https://en.wikipedia.org/wiki/Fibonacci_number)
-sequence to estimate stories.
+We like to start our sprints on Mondays and end our sprints on Fridays. Depending on the project, sprints may be 1 or 2 weeks long.
+
+When we plan the sprint on Monday, we define which stories will be worked on during the sprint. At that point, stories should also be estimated. We like to use the [Fibonacci](https://en.wikipedia.org/wiki/Fibonacci_number)
+sequence to estimate stories and we use an internal tool to do [blind estimation](https://www.ombulabs.com/blog/agile/scrum/become-better-estimators.html) as we are a remote, distributed team.
 
-If a story is more complicated than 8 points, you should break it down in two or
-more stories. When you estimate, you must remember that you are [estimating
-complexity](https://rubygarage.org/blog/3-reasons-to-estimate-with-story-points),
-**not hours**. Different developers might take different amount of time
-in completing that story, so **we shouldn't estimate hours**.
+It is also important to keep in mind that when estimating stories, we are [estimating complexity](https://rubygarage.org/blog/3-reasons-to-estimate-with-story-points), **not hours**. Different developers might take different amount of time in completing that story, so **we shouldn't estimate hours**.
 
-For the example we mentioned before, we estimate that that story has 3 points of
-complexity.
+Story points are not hours. Given a user story that has 3 story points of complexity, a Senior Developer might take 2 hours shipping it, a Junior Developer might take 5 hours shipping it.
 
-<img src="/blog/assets/images/user-story-estimation-fibonacci.png" alt="State of a simple user story before you estimate it" class="medium-img">
+This will affect metrics such as team velocity, which are important when planning sprints. So it is always important to remember to think in terms of story points, not hours.
 
-Story points are not hours. Given a user story that has 3 story points of complexity,
-a Senior Developer might take 2 hours shipping it, a Junior Developer might take
-5 hours shipping it.
+Once stories are estimated, the team is ready to kick-off the sprint.
 
-## Step 4: Starting and finishing user stories
+## Step 4: Starting user stories
 
-You should always pick the first unassigned story from the top of the backlog.
-If the first story is assigned to someone, you may check with them if they need
-your help with it.
+Once the sprint is planned, it's time to start working on the stories currently in the iteration.
 
-<img src="/blog/assets/images/user-story-start.png" alt="State of a user story after you have estimated it" class="medium-img">
+When a story is started, the clock starts ticking on that story, and we should try to finish it as soon as possible. In Pivotal Tracker, to indicate work on a story has been started, the *Start* button should be clicked.
 
```

Comment:
I'm not sure where, but maybe we can link the pivotal tracker help articles on some places, like this https://www.pivotaltracker.com/help/articles/story_states/ for more details on the states.

Reply:
Good point. I added an extra bit for that :)